Gemfile

```diff
@@ -1,2 +1,2 @@
-source 'https://rubygems.org'
+source 'https://rubygems.org/'
```

Comment:
I'd changed it to the Asia specific server for rubygems because rubygems.org was down momentarily or something. When I pushed the commit I think I left a '/' there by mistake. 
Will edit this too.